rust/stackablectl/src/cmds/release.rs

```diff
@@ -319,6 +356,87 @@ async fn install_cmd(
     }
 }
 
+#[instrument(skip(cli, release_list, transfer_client), fields(indicatif.pb_show = true))]
```

Comment:
We should really `skip_all` and just allow the fields we want to appear in spans.

```suggestion
#[instrument(skip_all, fields(indicatif.pb_show = true))]
```